src/pages/GameLobby/GameLobby.js

```diff
@@ -0,0 +1,139 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import PlayerAvatar from "../../components/PlayerAvatar";
+import {
+  AppBar,
+  Dialog,
+  DialogTitle,
+  DialogContentText,
+  DialogContent,
+  DialogActions,
+  Button,
+  Slide
+} from "@material-ui/core";
+
+function Transition(props) {
+  return <Slide direction="up" {...props} />;
+}
+const styles = theme => ({
+  root: {},
+  row: {
+    minHeigh: "16vh",
+    maxHeight: "16vh",
+    height: "16vh",
+    margin: "1vh",
+    padding: "1vh"
+  },
+  bar: {
+    height: "5vh"
+  },
+  textBox: {
+    margin: "2vh"
+  }
+});
+
+class GameLobby extends Component {
+  constructor() {
+    super();
+    this.state = {
+      startGameDialogOpen: false,
+      decks: []
+    };
+    this.decks = [];
+
+    this.getDeck();
+  }
+
+  startGame = index => {
+    this.setState({ startGameDialogOpen: true });
+    console.log(`Deck selected ${index}`);
+  };
+
+  cancelStartGame = () => {
+    this.setState({ startGameDialogOpen: false });
+  };
+
+  getDeck = () => {
+    this.decks = [
+      { id: 1, name: "Basic deck", JobId: 1 },
+      { id: 1, name: "Basic deck", JobId: 1 },
+      { id: 1, name: "Basic deck", JobId: 1 },
+      { id: 1, name: "Basic deck", JobId: 1 },
+      { id: 1, name: "Basic deck", JobId: 1 },
+      { id: 1, name: "Basic deck", JobId: 1 }
+    ];
+  };
+
+  generateRow = (decks, index) => {
+    let grid = [];
+    if (decks.length > index) {
+      grid.push(
+        <Grid item md>
+          <PlayerAvatar
+            clickAction={this.startGame}
+            index={index}
+            tag={decks[index].name}
+          />
+        </Grid>
+      );
+    }
+    if (decks.length > index + 1) {
+      grid.push(
+        <Grid item md>
+          <PlayerAvatar
+            clickAction={this.startGame}
+            index={index + 1}
+            tag={decks[index + 1].name}
+          />
+        </Grid>
+      );
+    }
+    return grid;
+  };
+
+  render() {
+    const { token, classes } = this.props;
+    const { startGameDialogOpen } = this.state;
+    return (
+      <div>
+        <AppBar position="static" cltokenassName={classes.bar}>
+          <div>Back button and stuff</div>
+        </AppBar>
+        <div className={classes.textBox}>Select deck to start game with :</div>
+        <Grid container>
```

Comment:
Je pense que je ferai un composant a part pour la grid de deck. (`DeckList` ?) Je metterai comme props `cards` `width` `height` et je ferais deux boucle pour générer les colonnes et les lignes pour te deck (dans le cas actuel, width=2, height=3 je pense ?)

Reply:
The max deck for now is 6 and the width will always be 2. That page is just a place holder I need to be able to test the game board with the server. I'll add an issue to scrap and redo it.

Reply:
Added : https://github.com/ay-project/ay-project-frontend-react/issues/11